clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/actions/DestroyTitusJob.java

```diff
@@ -68,7 +70,7 @@ public Result apply(@NotNull DestroyTitusJob.DestroyTitusJobCommand command, @No
         titusClientProvider.getTitusClient(
             (NetflixTitusCredentials) accountCredentials, command.description.getRegion());
 
-    Job job = titusClient.getJobAndAllRunningAndCompletedTasks(command.description.getJobId());
+    Job job = fetchJob(titusClient, command.description.getJobId());
```

Comment:
The titus client has differing behaviors depending on method called. 

Some will return null, some will raise the `StatusRuntimeException`. 

It's unlikely to happen all that often in practice but this was a gap that existed before this was converted to a `SagaAction`.